content/articles/switch-vs-router-vs-hub/index.md

```diff
@@ -0,0 +1,117 @@
+In networking the terms "Switches", "Hubs", and "Routers" are sometimes used interchangeably and this is wrong. Despite them being similar, there are differences in how they handle data. These three components may be integrated into a single device making it hard for a student to distinguish between them.
+
+In this article, we are going to discuss each device and their functionalities in a network.
+
+### Switch

```

Comment:
Please add a "Table of contents" section for the reader in between the sentence and the header. It is important to always include one inside your article. Also make sure they link to their respective headers